transports/azure-storage-queues/delayed-delivery_disabling_asqn_[9,].partial.md

```diff
@@ -0,0 +1,10 @@
+
+### Disabling delayed delivery
```

Comment:
This section is identical between versions 8 and 9. What are the benefits of splitting it into the dedicated partial for v9?

Reply:
This is needed because of the new Nuget alias. 👍